server/src/user/service.js

```diff
@@ -28,11 +33,14 @@ exports.add = (req, res) => {
   })
 
   newUser.save(newUser)
-    .then(() => res.json(201, {
-      success: true,
-      message: res.__('success.add'),
-      user: newUser
-    }))
+    .then(() => {
+      mailer.send(newUser.email, 'User created', res.__('mail.activation', 'http://localhost:8081/users/activation/' + newUser.token))
```

Comment:
_User created_ should be localized as well...you can put in `email.subject.created`

Reply:
res.json(201, {
        success: true,
        message: res.__('success.add'),
        user: newUser
      })

it is already localized :)

Reply:
How is the second parameter of send localized? I see it hard coded 'User created'.

Reply:
i don't know what are you talking about. The message is localized :)

Reply:
When I put a comment on a line, you should check the line where I put the comment not the previous lines... I'm talking about line 37 : to be precise this line: 
```
mailer.send(newUser.email, 'User created', res.__('mail.activation', 'http://localhost:8081/users/activation/' + newUser.token))
``` 
the second param is not localized, I suspect that is the email subject...